### DISCUSSION TITLE
wire:loading.class.delay produces errors in console when clearing (search) field

### DISCUSSION THREAD

**Discussion by regrub** [livewire/livewire#5656] [7. Bugs (archived)]
< td wire:loading.class.delay="opacity-50">

<img width="530" alt="Scherm­afbeelding 2023-04-16 om 11 12 28" src="https://user-images.githubusercontent.com/49644016/232289102-8aa6ca31-af7a-4035-9343-1a626a937385.png">

**Comment by joshhanley** [comment]
@regrub thanks for reporting this issue. Can you provide a Livewire component class and view, that demonstrates the issue with as little code as possible, so we can investigate? Thanks!
